apartment_application_service/urls.py

```diff
@@ -1,14 +1,13 @@
 from django.urls import include, path
 from helusers.admin_site import admin
-from rest_framework import routers
 
 from application_form import urls as api_urls
-
-router = routers.DefaultRouter()
+from connections import urls as rpc_api_urls
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("v1/", include(api_urls, namespace="v1")),
+    path("v1/", include((api_urls, "application_form"), namespace="v1")),
+    path("v1/", include((rpc_api_urls, "connections"))),
```

Comment:
Should we set these to own namespace e.g. `v1/connections/` and `v1/applications` ? Later it will be much harder to make changes to these as the other systems have been integrated into this. Could be also that it will never happen.